pkg/Cpanel/Security/Advisor/Assessors/Mysql.pm

```diff
@@ -109,6 +113,32 @@ sub _check_for_anonymous_users {
     return 1;
 }
 
+sub _check_for_public_bind_address {
+    my $self = shift;
+
+    my $mycnf        = Cpanel::MysqlUtils::MyCnf::Full::etc_my_cnf();
+    my $bind_address = $mycnf->{'mysqld'}->{'bind-address'};
+
+    my @deny_rules   = grep { /--dport 3306/ && /-j (DROP|REJECT)/ } split /\n/, Cpanel::SafeRun::Errors::saferunnoerror( '/sbin/iptables',  '--list-rules' );
```

Comment:
Suggestion: Since we've already loaded my.cnf, we could also incorporate the `port` option under `[mysqld]`.